charts/erzwingungshaft/templates/secret.yaml

```
@@ -0,0 +1,20 @@
+{{- $dot := . }}
+{{- range $secret := .Values.secrets -}}
+{{ $data := dict "dot" $dot "module" $secret }}
+{{- if not (lookup "v1" "Secret" $dot.Release.Namespace $secret.name) }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)